aws-android-sdk-mobile-client/src/androidTest/java/com/amazonaws/mobile/client/AWSMobileClientCustomAuthTest.java

```diff
@@ -35,8 +35,11 @@
 import java.util.HashMap;
 import java.util.Map;
 import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
 
 import static com.amazonaws.mobile.auth.core.internal.util.ThreadUtils.runOnUiThread;
+import static com.amazonaws.mobile.client.AWSMobileClientPersistenceTest.PASSWORD;
```

Comment:
For future cleanup, can we put the user & password credentials in the same file?